pyaedt/generic/constants.py

```diff
@@ -528,19 +546,19 @@ class SolverType(object):
 
 
 class CutoutSubdesignType(object):
-    (Conformal, BoundingBox) = range(0, 2)
+    (Conformal, BoundingBox, Invalid) = range(0, 3)
 
 
 class RadiationBoxType(object):
-    (Conformal, BoundingBox, ConvexHull) = range(0, 3)
+    (Conformal, BoundingBox, ConvexHull, Invalid) = range(0, 4)
 
 
 class SweepType(object):
-    (Linear, LogCount) = range(0, 2)
+    (Linear, LogCount, Invalid) = range(0, 3)
 
 
 class BasisOrder(object):
-    (Mixed, Zero, single, Double) = range(0, 4)
+    (Mixed, Zero, single, Double, Invalid) = range(0, 5)
```

Comment:
@maxcapodi78 I am assuming it is too late to rename `single` as `Single`.
Could this cause deprecation issue?
```suggestion
    (Mixed, Zero, Single, Double, Invalid) = range(0, 5)
```

Reply:
@maxcapodi78  I think this is a typo and can be fixed since PR is not yet closed. Correct @isaacansys ?

Reply:
@maxcapodi78 I did not change this so it was in the previous commit as well.  This value with lowercase is already used in EDB_Data.py for example.

Reply:
If customers/users are already using it. We can let it the way it is. I was mentioning "deprecation" because that it could require to update some scripts and examples.

Reply:
@MaxJPRey after discussion with @maxcapodi78 we think we should make this change now as this was recently added and we can make things consistent with minimum pain now.  Should just make sure we announce this so existing users can adjust.  Thoughts?

Reply:
Please add a warning into the class docstrings

Reply:
Agreed.